oeps/processes/oep-0055/decisions/0001-use-backstage-to-support-maintainers.rst

```diff
@@ -0,0 +1,126 @@
+0001 Use Backstage to Support Maintainers
+#########################################
+
+Status
+******
+
+**Draft**
+
+.. TODO Upon Acceptance
+
+	* Update OEP to mention the catalog-info.ymal file instead of the readme.
+		* Relevant link https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055-proc-project-maintainers.html#documenting-maintainers
+
+Context
+*******
+
+* :doc:`OEP-55 <../../oep-0055-proc-project-maintainers>` laid out that we should document the maintainers of a component.
+
+* As a maintainer it is hard to know what components you are maintaining.
+
+* As a community member it is currently difficult to quickly look up who the
+  maintainers are for multiple components quickly.
+
+* Given the number of repos that make up the Open edX platform and the
+  complexity of the job of maintaining them, we need a system for cataloging,
+  and making transparent the critical details of our repositories.
+
+Decision
+********
+
+To solve the above issues and to enable the potential to ease other
+discoverability and maintainership burdens, the Open edX community shall
+maintain a `backstage`_ instance and related metadata.
+
+Consequences
```

Comment:
>     2U/edX were maintaining much of this information in a spreadsheet before the tCRIL-split. This ADR could acknowledge the need to update this existing tooling. Even if not mentioned in the ADR, this work needs to be communicated and planned for.

That feels like a consequence internal to 2U and not one for the open source project.  I agree the work needs to be done either way, but I'm not sure mentioning it here is valuable from the open source project perspective.

>     The existing spreadsheet also has exceptional maintenance requirements for the edx-platform monolith. Does tCRIL plan on supporting that level of detail, or is it planning on having edx-platform maintained by 2U/edX, with the split ownership being decided/documented/maintained by 2U/edX as we see fit?

I think edx-platform maintainership is being avoided for the pilot but we will eventually need to make a plan for it.  Given its complexity, I expect there will be a future ADR that addresses its ownership specifically.

Reply:
@robrap maybe we can touch base over slack about the nature of the maintenance requirements? I'm curious how much of it would be relevant to any organization maintaining `edx-platform` vs 2U specifically.